doc/template.html

```diff
@@ -1,57 +1,94 @@
-              <a href="all.html">View on single page</a>
-            </li>
-            <li>
-              <a href="__FILENAME__.json">View as JSON</a>
-            </li>
-            __ALTDOCS__
-            __EDIT_ON_GITHUB__
-          </ul>
-        </div>
-        <hr>
-      </header>
+      <div id="column1" data-id="__ID__" class="interior">
+        <header>
+          <div class="header-container">
+            <h1>Node.js __VERSION__ Documentation</h1>
+            <button class="theme-toggle-btn" id="theme-toggle-btn">
+              <span class="icon dark-icon"
+                ><svg xmlns="http://www.w3.org/2000/svg" height="24" width="24">
+                  <path fill="none" d="M0 0h24v24H0z" />
+                  <path
+                    d="M11.1 12.08c-2.33-4.51-.5-8.48.53-10.07C6.27 2.2 1.98 6.59 1.98 12c0 .14.02.28.02.42.62-.27 1.29-.42 2-.42 1.66 0 3.18.83 4.1 2.15A4.01 4.01 0 0111 18c0 1.52-.87 2.83-2.12 3.51.98.32 2.03.5 3.11.5 3.5 0 6.58-1.8 8.37-4.52-2.36.23-6.98-.97-9.26-5.41z"
+                  />
+                  <path
+                    d="M7 16h-.18C6.4 14.84 5.3 14 4 14c-1.66 0-3 1.34-3 3s1.34 3 3 3h3c1.1 0 2-.9 2-2s-.9-2-2-2z"
+                  /></svg
+              ></span>
+              <span class="icon light-icon"
+                ><svg xmlns="http://www.w3.org/2000/svg" height="24" width="24">
+                  <path d="M0 0h24v24H0z" fill="none" />
+                  <path
+                    d="M6.76 4.84l-1.8-1.79-1.41 1.41 1.79 1.79 1.42-1.41zM4 10.5H1v2h3v-2zm9-9.95h-2V3.5h2V.55zm7.45 3.91l-1.41-1.41-1.79 1.79 1.41 1.41 1.79-1.79zm-3.21 13.7l1.79 1.8 1.41-1.41-1.8-1.79-1.4 1.4zM20 10.5v2h3v-2h-3zm-8-5c-3.31 0-6 2.69-6 6s2.69 6 6 6 6-2.69 6-6-2.69-6-6-6zm-1 16.95h2V19.5h-2v2.95zm-7.45-3.91l1.41 1.41 1.79-1.8-1.41-1.41-1.79 1.8z"
+                  /></svg
+              ></span>
+            </button>
+          </div>
+          <div id="gtoc">
+            <ul>
+              <li>
+                <a href="index.html">Index</a>
+              </li>
+              <li>
+                <a href="all.html">View on single page</a>
+              </li>
+              <li>
+                <a href="__FILENAME__.json">View as JSON</a>
+              </li>
+              __ALTDOCS__ __EDIT_ON_GITHUB__
+            </ul>
+          </div>
+          <hr />
+        </header>
 
-      <div id="toc">
-        <h2>Table of Contents</h2>
-        __TOC__
-      </div>
+        <div id="toc">
+          <h2>Table of Contents</h2>
+          __TOC__
+        </div>
 
-      <div id="apicontent">
-        __CONTENT__
-        <!-- API END -->
+        <div id="apicontent">
+          __CONTENT__
+          <!-- API END -->
+        </div>
       </div>
     </div>
-  </div>
-</body>
+  </body>
+  <script>
+    function init() {
+      if (
+        window.matchMedia &&
+        window.matchMedia("(prefers-color-scheme: dark)").matches
+      ) {
+        document.body.classList.add("dark-mode");
```

Comment:
I think it would make sense to listen for preference changes as well:
```suggestion
      if (window.matchMedia) {
        const mq = window.matchMedia("(prefers-color-scheme: dark)");
        md.addEventListener("change", function(e){
          document.body.classList.toggle("dark-mode", e.matches);
        })
        mq.matches && document.body.classList.add("dark-mode");
```